__tests__/app.test.js

```diff
@@ -298,3 +298,32 @@ describe("PATCH /api/articles/:article_id", () => {
       });
   });
 });
+
+describe("DELETE /api/comments/:comment_id", () => {
+  test("should delete a comment from a given comment_id ", () => {
+    return request(app)
+      .delete("/api/comments/1")
+      .expect(204)
+      .then(({ body }) => {
+        expect(body).toEqual({});
+      });
```

Comment:
You don't need to check the body - a delete endpoint will never send a body so just the 204 is enough!